fieldservice/models/fsm_order.py

```diff
@@ -12,13 +12,15 @@ class FSMOrder(models.Model):
     _inherit = ['mail.thread', 'mail.activity.mixin']
 
     def _default_stage_id(self):
-        return self.env.ref('fieldservice.fsm_stage_new')
+        return self.team_id.stage_ids[0]
 
+    team_id = fields.Many2one('fsm.team', string='Service Team')
```

Comment:
@dreispt We want a default team and the team field on the order views should be seen only by the group_fsm_team group.